modules/database/database.tf

```diff
@@ -0,0 +1,21 @@
+resource "aws_dynamodb_table" "database" {
+  hash_key = "company_name"
```

Comment:
We can put like glasslense_company or other similar.

Reply:
Same as the database name. I left generic so we could reuse it later

Reply:
Nice.We can keep it